src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -39,6 +39,7 @@ import type {Multitasking} from '../../notification/NotificationRepository';
 import Duration from './Duration';
 import NamedIcon from 'Components/NamedIcon';
 import ButtonGroup from 'Components/ButtonGroup';
+import type {Participant} from 'src/script/calling/Participant';
```

Comment:
```suggestion
import type {Participant} from './Participant';
```

Reply:
participant is not there, this is inside components/calling...